cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -200,50 +101,275 @@
             }
         });
     }
+
+    // Expandable Overrides
```

Comment:
Added in https://github.com/cfpb/cfgov-refresh/commit/3f65fc6c860c9e8fd00f10a506ea055b52a8b946.